code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -1,7 +1,55 @@
-#define CRYOMOBS 'icons/obj/cryo_mobs.dmi'
 ///Max temperature allowed inside the cryotube, should break before reaching this heat
 #define MAX_TEMPERATURE 4000
 
+/// This is a visual helper that shows the occupant inside the cryo cell.
+/atom/movable/cryo_occupant_vis
+	icon = 'icons/obj/cryogenics.dmi'
+	// Must be tall, otherwise the filter will consider this as a 32x32 tile
+	// and will crop the head off.
+	icon_state = "mask_bg"
+	layer = ABOVE_WINDOW_LAYER + 0.01
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	var/obj/machinery/atmospherics/components/unary/cryo_cell/cryo
+	var/animation = FALSE
+	pixel_y = 22
+	appearance_flags = KEEP_TOGETHER
+
+/atom/movable/cryo_occupant_vis/New(obj/machinery/atmospherics/components/unary/cryo_cell/cryo_)
+	cryo = cryo_
+	return ..()
```

Comment:
Do this in Initialize()

Reply:
Does the parameters used with `new()` transfer to `Initialize` as well?